api/src/main/scala/za/co/absa/loginsvc/utils/AwsSecretsUtils.scala

```diff
@@ -66,3 +66,12 @@ object AwsSecretsUtils {
     }
   }
 }
+
+trait SecretUtil {
```

Comment:
```suggestion
trait SecretUtils {
```

suggesting a rename to follow the suite of the object name